plugins/grafana-custom-plugins/grafana-http-table-plugin/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./.config/tsconfig.json"
+}
```

Comment:
There are lots of generated file missing empty line. Do we have a method to avoid that? Or we should add it manually?

Reply:
missing empty line